proto/src/determined/master/v1/master.proto

```diff
@@ -55,16 +55,17 @@ enum ResourceAllocationAggregationType {
   RESOURCE_ALLOCATION_AGGREGATION_TYPE_LABEL = 3;
 }
 
-// One instance of slots in the cluster being allocated to a task during a day
-// (aggregated).
+// One instance of slots in the cluster being allocated to a task during a
+// period (aggregated).
 message ResourceAllocationAggregatedEntry {
```

Comment:
Took me a second to grok this API, probably a lot better for a user I agree, but dang is it a lot funkier on the backend. Immediately think of https://en.wikipedia.org/wiki/Object%E2%80%93relational_impedance_mismatch.

Reply:
Heh, yeah; I hadn't heard that term, but that's basically exactly how I felt. I tell myself it's our job to do the funky stuff so everything comes out nice.